h/security/predicates.py

```diff
@@ -143,12 +143,32 @@ def group_created_by_user(identity, context):
 
 
 @requires(authenticated_user, group_found)
-def group_has_user_as_member(identity, context):
-    # With detached groups like we have with the websocket, this doesn't work
-    # as SQLAlchemy does not consider them equal:
-    # return context.group in identity.user.groups
```

Comment:
This was a distracting comment giving a commented-out alternative implementation of the line below it, plus a comment explaining why the commented-out version wouldn't work. I just removed the comment.